### PR TITLE
Bug/added srcset to split two column images

### PR DESCRIPTION
**Pull request by koyan**

### CODE REVIEW

File: includes/blocks/split_two_columns.twig [hunk 1]
```diff
@@ -31,9 +31,13 @@
 		<div class="split-two-column-item item--right">
 			{% if ( campaign.image ) %}
 				{% if ( campaign.focus ) %}
-					<div class="split-two-column-item-image" style="background-image:url({{ campaign.image }}); background-position:{{ campaign.focus }};"></div>
+					<div class="split-two-column-item-image" style="background-position:{{ campaign.focus }};">
+						<img src="{{ campaign.image }}" srcset="{{ campaign.srcset }}">
```
Comment (sagarsdeshmukh) [inline includes/blocks/split_two_columns.twig]: Do you think we need alt attribute here ?

Comment (koyan) [reply includes/blocks/split_two_columns.twig]: Correct. Adding it now

File: includes/blocks/split_two_columns.twig [hunk 2]
```diff
@@ -31,9 +31,13 @@
 		<div class="split-two-column-item item--right">
 			{% if ( campaign.image ) %}
 				{% if ( campaign.focus ) %}
```
Comment (angtheod) [inline includes/blocks/split_two_columns.twig]: Should we have a default value for campaign.focus? If image is set but focus is not?

Comment (koyan) [reply includes/blocks/split_two_columns.twig]: No, if none is given by the editor, the center is used by default by the css

Comment (angtheod) [reply includes/blocks/split_two_columns.twig]: oh yea.. and there is an else to that if... didn't see it.. sorry

**Pull request merged**